corehq/blobs/migrate_metadata.py

```diff
@@ -267,7 +293,7 @@ def get_shared_domain(doc):
 
 def get_invoice_domain(doc):
     if doc.get("is_wire"):
-        return acct.WireInvoice.objects.get(int(doc["invoice_id"])).domain
+        return acct.WireInvoice.objects.get(id=int(doc["invoice_id"])).domain
```

Comment:
I know this wasn't changed in this PR, but I was curious about the ```import x.models as y``` pattern used here.  Was there any particular reason you chose to use that instead of the more commonly used ```from x.models import <model names>```?

Reply:
I can't remember exactly, but I think I did it because `acct` references a module and not an object within a module. A module would normally be imported like
```py
import package.module
```
but in this case I wanted to reference it with a shorter name than `package.module`, so I changed that to
```py
import package.module as mod`
```
I imported the module rather than the names referenced from within the module for convenience and to keep this module's namespace a bit cleaner.